runtime/rt.c

```diff
@@ -0,0 +1,69 @@
+#include <stdio.h>
```

Comment:
Why is this file in c now? We can't use runtime/src/main.rs?

Reply:
We need both c and rust. The bril repo has the `.rs` runtime for brillvm and `.c` runtime for brilift. See `src/util.rs`.

Reply:
Oh, I see. Thank you